auslib/migrate/versions/016_add_shutoffs.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Table, Column, Integer, String, MetaData, UniqueConstraint
+
+metadata = MetaData()
+
+shutoffs = Table(
+    "shutoffs", metadata,
+    Column('shutoff_id', Integer, primary_key=True, autoincrement=True),
+    Column("product", String(15), nullable=False),
+    Column("channel", String(75)),
+    Column("mapping", String(100)),
+    UniqueConstraint('product', 'channel')
+)
+
+
+def upgrade(migrate_engine):
+    metadata.bind = migrate_engine
+    metadata.create_all()
+
+
+def downgrade(migrate_engine):
+    metadata.bind = migrate_engine
```

Comment:
I don't think this downgrade method does what you want...it appears to be creating the table instead of dropping it!